job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1660,6 +1660,29 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /**
+     * Generates trend report for SLOCCount and cloc.
+     *
+     * @since 1.41
+     */
+    @RequiresPlugin(id = 'sloccount', minimumVersion = '1.20')
+    void sloccount(@DslContext(SlocCountContext) Closure closure) {
+        SlocCountContext context = new SlocCountContext()
+        ContextHelper.executeInContext(closure, context)
+
+        publisherNodes << new NodeBuilder().'hudson.plugins.sloccount.SloccountPublisher' {
+            if (context.pattern != '') {
```

Comment:
Instead of the `if`, use `pattern(context.pattern ?: '')'  to deals with null value. It's OK to create an empty element.